src/coreclr/jit/objectalloc.h

```diff
@@ -181,6 +254,11 @@ inline bool ObjectAllocator::CanAllocateLclVarOnStack(unsigned int         lclNu
         return false;
     }
```

Comment:
It seems like we should have a limit on the aggregate stack allocated size. That's somewhat preexisting, but probably more important now.